qa/tasks/cephfs/mount.py

```diff
@@ -14,6 +14,32 @@
 
 log = logging.getLogger(__name__)
 
+# This will cleanup the stale netnses, which are from the
+# last failed test cases.
+def cleanup_stale_netnses_and_bridge(remote):
+    p = remote.run(args=['ip', 'netns', 'list'],
+                   stdout=StringIO(), timeout=(5*60))
+    p = p.stdout.getvalue().strip()
+
+    # Get the netns name list
+    netns_list = re.findall(r'ceph-ns-[^()\s][-.\w]+[^():\s]', p)
+
+    # Remove the stale netnses
+    for ns in netns_list:
+        ns_name = ns.split()[0]
+        args = ['sudo', 'ip', 'netns', 'delete', '{0}'.format(ns_name)]
+        try:
+            remote.run(args=args, timeout=(5*60))
+        except Exception:
+            pass
+
+    # Remove the stale 'ceph-brx'
+    try:
+        args = ['sudo', 'ip', 'link', 'delete', 'ceph-brx']
+        remote.run(args=args, timeout=(5*60))
```

Comment:
Set omit_sudo to True.

Reply:
You mean we should set it to False, right ?

We are fixing this in PR#35944 together with defer deleting the netns. In that PR we are using the run_shell_payload() and have fixed it :-)

Thanks @rishabh-d-dave

Reply:
Yes, I should've written "Set `omit_sudo` to `False`".